### PR TITLE
[DOCS] Change PHP Template Examples to Twig

### LINKED ISSUE #7229: [DOCS] Change PHP Template Examples to Twig, Remove PHP Templates

### PR DESCRIPTION
**Pull request by brusch**
Resolves #7229

### CODE REVIEW

File: doc/Development_Documentation/10_E-Commerce_Framework/15_Payment/10_Recurring_Payments.md [hunk 1]
```diff
@@ -56,59 +57,48 @@ public function paymentAction(Request $request)
...
-            endif;
-        endforeach;
-        ?>
+<form method="post" action="{{ pimcore_url({action: 'confirm'}, 'checkout', true) }}">
+
+    {% if sourceOrders is not empty %}
+
+        <h4>{{ 'checkout.use-recurring-payment'|trans }}</h4>
+
+        {% for paymentMethod in paymentMethods %}
```
Comment (aarongerig) [inline doc/Development_Documentation/10_E-Commerce_Framework/15_Payment/10_Recurring_Payments.md:80]:
```suggestion
        {% for paymentMethod in paymentMethods if sourceOrders[paymentMethod] is defined %}
```

File: doc/Development_Documentation/10_E-Commerce_Framework/15_Payment/10_Recurring_Payments.md [hunk 2]
```diff
@@ -56,59 +57,48 @@ public function paymentAction(Request $request)
...
+            {% set sourceOrder = sourceOrders[paymentMethod] %}
+            {% if sourceOrder %}
+                {% set paymentProvider = sourceOrder.paymentProvider.paymentProviderQpay %}
```
Comment (aarongerig) [inline doc/Development_Documentation/10_E-Commerce_Framework/15_Payment/10_Recurring_Payments.md:83]:
```suggestion
                {% set paymentProvider = sourceOrders[paymentMethod].paymentProvider.paymentProviderQpay %}
```

File: doc/Development_Documentation/10_E-Commerce_Framework/15_Payment/10_Recurring_Payments.md [hunk 3]
```diff
@@ -56,59 +57,48 @@ public function paymentAction(Request $request)
...
+                {% if paymentProvider %}
+                    {% currentPaymentMethod = paymentProvider.Auth_paymentType %}
+                     <p>
+                        <input name="recurring-payment" value="{{ sourceOrder.id }}" type="radio">
```
Comment (aarongerig) [inline doc/Development_Documentation/10_E-Commerce_Framework/15_Payment/10_Recurring_Payments.md:87]:
```suggestion
                        <input name="recurring-payment" value="{{ sourceOrders[paymentMethod].id }}" type="radio">
```

File: doc/Development_Documentation/10_E-Commerce_Framework/15_Payment/10_Recurring_Payments.md [hunk 4]
```diff
@@ -56,59 +57,48 @@ public function paymentAction(Request $request)
...
+
+    {% if sourceOrders is not empty %}
+
+        <h4>{{ 'checkout.use-recurring-payment'|trans }}</h4>
+
+        {% for paymentMethod in paymentMethods %}
+            {% set sourceOrder = sourceOrders[paymentMethod] %}
+            {% if sourceOrder %}
+                {% set paymentProvider = sourceOrder.paymentProvider.paymentProviderQpay %}
+                {% if paymentProvider %}
```
Comment (aarongerig) [inline doc/Development_Documentation/10_E-Commerce_Framework/15_Payment/10_Recurring_Payments.md:84]:
```suggestion

```

File: doc/Development_Documentation/10_E-Commerce_Framework/15_Payment/10_Recurring_Payments.md [hunk 5]
```diff
@@ -56,59 +57,48 @@ public function paymentAction(Request $request)
...
-        ?>
+<form method="post" action="{{ pimcore_url({action: 'confirm'}, 'checkout', true) }}">
+
+    {% if sourceOrders is not empty %}
+
+        <h4>{{ 'checkout.use-recurring-payment'|trans }}</h4>
+
+        {% for paymentMethod in paymentMethods %}
+            {% set sourceOrder = sourceOrders[paymentMethod] %}
+            {% if sourceOrder %}
```
Comment (aarongerig) [inline doc/Development_Documentation/10_E-Commerce_Framework/15_Payment/10_Recurring_Payments.md:82]:
```suggestion

```

File: doc/Development_Documentation/10_E-Commerce_Framework/15_Payment/10_Recurring_Payments.md [hunk 6]
```diff
@@ -56,59 +57,48 @@ public function paymentAction(Request $request)
...
+                        <strong>{{ currentPaymentMethod }}</strong>
+                        
+                        {% if currentPaymentMethod === 'SEPA-DD' %}
+                            {{ paymentProvider.Auth_bankAccountOwner }} {{ paymentProvider.Auth_bankAccountIBAN }}
+                        {% elseif currentPaymentMethod === 'CCARD' %} 
+                            {{ paymentProvider.Auth_maskedPan }} {{ paymentProvider.Auth_expiry }}
+                        {% endif %}
+                    </p>
+                {% endif %}
+            {% endif %}
```
Comment (aarongerig) [inline doc/Development_Documentation/10_E-Commerce_Framework/15_Payment/10_Recurring_Payments.md:97]:
```suggestion

```

File: doc/Development_Documentation/10_E-Commerce_Framework/15_Payment/10_Recurring_Payments.md [hunk 7]
```diff
@@ -56,59 +57,48 @@ public function paymentAction(Request $request)
...
+            {% set sourceOrder = sourceOrders[paymentMethod] %}
+            {% if sourceOrder %}
+                {% set paymentProvider = sourceOrder.paymentProvider.paymentProviderQpay %}
+                {% if paymentProvider %}
+                    {% currentPaymentMethod = paymentProvider.Auth_paymentType %}
+                     <p>
+                        <input name="recurring-payment" value="{{ sourceOrder.id }}" type="radio">
+                        <strong>{{ currentPaymentMethod }}</strong>
+                        
+                        {% if currentPaymentMethod === 'SEPA-DD' %}
```
Comment (aarongerig) [inline doc/Development_Documentation/10_E-Commerce_Framework/15_Payment/10_Recurring_Payments.md:90]:
```suggestion
                        {% if currentPaymentMethod is same as('SEPA-DD') %}
```

File: doc/Development_Documentation/10_E-Commerce_Framework/15_Payment/10_Recurring_Payments.md [hunk 8]
```diff
@@ -56,59 +57,48 @@ public function paymentAction(Request $request)
...
+                            {{ paymentProvider.Auth_bankAccountOwner }} {{ paymentProvider.Auth_bankAccountIBAN }}
+                        {% elseif currentPaymentMethod === 'CCARD' %}
```
Comment (aarongerig) [inline doc/Development_Documentation/10_E-Commerce_Framework/15_Payment/10_Recurring_Payments.md:92]:
```suggestion
                        {% elseif currentPaymentMethod is same as('CCARD') %} 
```

**Comment by brusch** [comment]
@aarongerig Thanks! 👍

File: doc/Development_Documentation/10_E-Commerce_Framework/15_Payment/10_Recurring_Payments.md [hunk 9]
```diff
@@ -56,59 +57,46 @@ public function paymentAction(Request $request)
...
-            endif;
-        endforeach;
-        ?>
+<form method="post" action="{{ pimcore_url({action: 'confirm'}, 'checkout', true) }}">
+
+    {% if sourceOrders is not empty %}
+
+        <h4>{{ 'checkout.use-recurring-payment'|trans }}</h4>
+
+        {% for paymentMethod in paymentMethods if sourceOrders[paymentMethod] is defined %}
```
Comment (blankse) [inline doc/Development_Documentation/10_E-Commerce_Framework/15_Payment/10_Recurring_Payments.md:80]: @brusch Is not possible in Twig 3 anymore.

> Adding an if condition on a for tag is deprecated in Twig 2.10. Use a filter filter or an “if” condition inside the “for” body instead (if your condition depends on a variable updated inside the loop).

Comment (brusch) [reply doc/Development_Documentation/10_E-Commerce_Framework/15_Payment/10_Recurring_Payments.md:80]: Thanks! Fixed in 61cab12868afb0fe6bd08bbb21486ac9fef9c69b

Comment (aarongerig) [reply doc/Development_Documentation/10_E-Commerce_Framework/15_Payment/10_Recurring_Payments.md:80]: @brusch Sorry for the caused extra work. 🙈

**Pull request merged**